drivesystem

arlenisinoa opened the pull request.

Commit:
Adding Travis Markdown

Commit:
Adding basic gradle infraastructure. Adding Travis and some other fun
stuff.

admin4571 committed:
Delete .project

admin4571 committed:
Delete .classpath

Commit:
Adding gradle support

Commit:
Updating travis yaml name

Commit:
Updating perms on gradlew

Commit:
Adding gradle libraries

Commit:
Updating build.gradle

Commit:
Fixing code coverage and adding test folder

Commit:
Adding test folder

Commit:
Adding shippable to README

admin4571 committed:
Merge pull request #2 from frc4571/add-shippable

Adding shippable to README

Commit:
Adding library jars

Commit:
Merge branch 'master' of https://github.com/frc4571/CrateRobot.git

Commit:
Updating README

Commit:
Fixing the gradle script. Now we will REALLY build a jar. A fat one at
that.

Commit:
Fixing for travis build

Commit:
Fixing so fatjar always gets built

Commit:
Cleaning up build.gradle and adding CD to Travis. Woo!

Commit:
Cleaning up build.gradle

Commit:
Cleaning up travis.yml

Commit:
Cleaning up travis.yml

Commit:
Checking CD

Commit:
Checking CD2

Commit:
Add Contributors badge

Commit:
Merge branch 'master' of https://github.com/frc4571/CrateRobot.git

Commit:
Fixing README

Commit:
Really fixing README

Commit:
Adding space between badges

admin4571 reviewed (changes requested):
This is an incorrect pull request. you should be having a pull request to merge the drive subsystem into master, not the other way around

Commit:
Adding wercker.yml

Commit:
Adding wercker badge

Commit:
Removing wercker

Commit:
Adding few more relevant badges

Commit:
Badge cleanup

Commit:
Adding MIT License

Commit:
Updating badge

Commit:
initial commit, work in progress

inyongie committed:
GradleRIO setup process ongoing

inyongie committed:
adding GradleRIO integrated build.gradle.

Commit:
changing build.gradle according to pull request

Commit:
header comment changes

Commit:
Committing test class for CodeCov

Commit:
Updating testcase

Commit:
Adding gradle task for coverage reports

Commit:
CodeCov params

inyongie committed:
for PR comments.

inyongie committed:
Toast is loaded. Removing the libs/ jars, as Toast/GradleRIO dependency seems to contain these.

inyongie committed:
Adding build info to README.md

inyongie committed:
Changing base class to extend IterativeModule, which is Toast's version of IterativeRobot.

arjunrao87 committed:
Merge pull request #9 from frc4571/feature_gradlerio

Feature gradlerio

Commit:
Backing out codecov changes to fix build

Commit:
Fixing test

Commit:
Fixing test

Commit:
Adding Codecov token

Commit:
Adding Coveralls

Commit:
Add version

Commit:
Hate this commit, but gotta try

Commit:
Updating the way plugins are invoked

Commit:
Moving order

Commit:
Gradle dependency

Commit:
Triggering build

Commit:
Trigger build

Commit:
Cleaning dependencies and adding CONTRIBUTING

Commit:
Everybody ♥s markdown

Commit:
Everybody ♥s Markdown

Commit:
Adding Best practices badge

arjunrao87 committed:
Set theme jekyll-theme-cayman
